src/Announcements-Core/Announcer.class.st

```diff
@@ -27,16 +27,22 @@ Class {
 Announcer >> announce: anAnnouncement [
 
 	^ self isSuspended
-		  ifFalse: [
-			  | announcement |
-			  announcement := anAnnouncement asAnnouncement.
-
-			  "If the user required to prevent the current announcement, we stop here."
-			  (self preventedAnnouncements handlesAnnouncement: announcement) ifTrue: [ ^ self ].
-
-			  registry ifNotNil: [ registry deliver: announcement ].
-			  announcement ]
-		  ifTrue: [ storedAnnouncements ifNotNil: [ storedAnnouncements add: anAnnouncement ] ]
+		ifFalse: [
+			| announcement |
+			announcement := anAnnouncement asAnnouncement.
```

Comment:
Tests were broken if the return was placed before this sentence... they expected that if you announcer a class, it's intanced and you get the instance as return.